docs/src/multivariate.md

```diff
@@ -71,6 +71,9 @@ invcov(::Distributions.AbstractMvNormal)
 logdetcov(::Distributions.AbstractMvNormal)
 sqmahal(::Distributions.AbstractMvNormal, ::AbstractArray)
 rand(::AbstractRNG, ::Distributions.AbstractMvNormal)
+minimum(::Distributions.AbstractMvNormal)
+maximum(::Distributions.AbstractMvNormal)
+extrema(::Distributions.AbstractMvNormal)
```

Comment:
For some reason the docs for minimum/maximum are not showing. I widened the docstring in Univariates.jl to `d::Distributions`, so I would think it should. `extrema` is showing

Reply:
ah... I see I have a `Base.extrema(d::Distribution)`, but there is not fallback for `minimum` and `maximum`. Please advise.

Reply:
See https://github.com/JuliaStats/Distributions.jl/pull/1319/files#r626651609, the suggestion there should address it.